libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.service.ts

```diff
@@ -153,4 +153,32 @@ export class NationalRegistryXRoadService {
       }
     )
   }
+
+  async getFamily(
+    user: User,
+    nationalId: string,
+  ): Promise<NationalRegistryFamilyMemberInfo[] | null> {
+    const family = await this.nationalRegistryApiWithAuth(user)
+      .einstaklingarGetFjolskyldumedlimir({ id: nationalId })
+      .catch(this.handle404)
+
+    if (family?.einstaklingar) {
+      return family.einstaklingar.map((member) => {
+        return (
+          member && {
+            nationalId: member.kennitala,
+            fullName: member.fulltNafn ?? '',
+            genderCode: member.kynkodi.toString(),
+            address: {
+              streetName: member.adsetur?.heiti ?? '',
+              postalCode: member.adsetur?.postnumer ?? '',
+              city: member.adsetur?.stadur ?? '',
+              municipalityCode: null,
+            },
+          }
+        )
+      })
+    }
+    return null
+  }
```

Comment:
What is the reason for checking member?
Also I would argue that the signature of the function should just be the array, and it should return empty array if nothing is found.
```suggestion
  async getFamily(
    user: User,
    nationalId: string,
  ): Promise<NationalRegistryFamilyMemberInfo[]> {
    const family = await this.nationalRegistryApiWithAuth(user)
      .einstaklingarGetFjolskyldumedlimir({ id: nationalId })
      .catch(this.handle404)

    return (family?.einstaklingar || []).map((member) => ({
      nationalId: member.kennitala,
      fullName: member.fulltNafn ?? '',
      genderCode: member.kynkodi.toString(),
      address: {
        streetName: member.adsetur?.heiti ?? '',
        postalCode: member.adsetur?.postnumer ?? '',
        city: member.adsetur?.stadur ?? '',
        municipalityCode: null,
      },
    }))
  }
```

Reply:
Yeah that makes sense, thank you for this